ADAL/src/broker/ios/ADBrokerHelper.m

```diff
@@ -201,8 +201,15 @@ + (BOOL)canUseBroker
     
     if (![ADAppExtensionUtil isExecutingInAppExtension])
     {
+        NSString *checkScheme = ADAL_BROKER_SCHEME;
+        
+        if (@available(iOS 13.0, *))
+        {
+            checkScheme = ADAL_BROKER_NONCE_SCHEME;
+        }
+        
         // Verify broker app url can be opened
-        return [[ADAppExtensionUtil sharedApplication] canOpenURL:[[NSURL alloc] initWithString:[NSString stringWithFormat:@"%@://broker", ADAL_BROKER_SCHEME]]];
+        return [[ADAppExtensionUtil sharedApplication] canOpenURL:[[NSURL alloc] initWithString:[NSString stringWithFormat:@"%@://broker", checkScheme]]];
```

Comment:
is `msauthv3` exclusive to us? if not, do we want to check both `ADAL_BROKER_SCHEME` and `ADAL_BROKER_NONCE_SCHEME`?

Reply:
I'll try to get exclusive one. If not, change logic later. For now assume exclusive.